index.html

```diff
@@ -0,0 +1,163 @@
+        <nav class="desktop-menu">
+            <a href="#"><img src="Images/CreativeArt.png" alt="logo" id="logo1"></a>
+            <ul class="dm">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+           <li><a href="#" class="campagne">Campagne</a></li> 
+            </ul>
+          
+        </nav>
+        <div class="main-wrap">
+            <nav class="mobile-menu"> 
+                <a href="#"><img src="Images/menu.png" alt="mobile-menu"></a>
+            </nav>
+            <p class="hello">"Hello! Sharing world"</p>
+            <h1 class="title">Creative Art Summit 2022</h1>
+            <p id="discription">
+             A joyful celebration where people from more than 80 countries who believe in the value of sharing and openness and create positive change gather and share their thoughts will be held in Korea in October.
+                </p>
+                <div>
+                    <p class="date">2022.10.14 (Fri) ~ 17 (Mon)</p>
+                    <p class="locate">@ National Museum of Korea, Art Center Nabi, etc.
+                    </p>
+                </div>
+        </div>
+    </header>
+        <section id="main-program">
+            <h2 class="main-head">Main Program</h2>
+            <hr>
+            <div class="program">
+            <div id="main-info">
+                <img src="Images/program_icon_01.png" alt="#" class="icon1">
+                <h2 id="subtitle">Lecture</h2>
+                <p class="support">Listen to lectures from speakers from around the world and learn about the latest trends in the world.</p>
+            </div>
+            <div id="main-info1">
+                <img src="Images/program_icon_02.png" alt="#" class="icon2">
+                <h2 id="subtitle1">CA Exhibition</h2>
+                <p class="support1">Appreciate various artists from around the world as they display their artwork.</p>
+            </div>
+            <div id="main-info2">
+                <img src="Images/program_icon_03.png" alt="#" class="icon3">
+                <h2 id="subtitle2">Forum</h2>
+                <p class="support2">Have the time to share your thoughts and opinions with experts from each topic.</p>
+            </div>
+            <div id="main-info3">
+                <img src="Images/program_icon_04.png" alt="#" class="icon4">
+                <h2 id="subtitle3">Workspace</h2>
+                <p class="support3">Try creating your own work rather than just watching.</p>
+            </div>
+            <div id="main-info4">
+                <img src="Images/program_icon_05.png" alt="#" class="icon5">
+                <h2 id="subtitle4">CA Ignite</h2>
+                <p class="support4">Get opportunities to work with other artists from around the world, also after the summit.</p>
+            </div>
```

Comment:
- [ ] Based on project requirements, you should be using `Cocogoose, Lato` fonts, however now all the texts have `sans-serif` font family. Please consider adding a proper font family 👍🏻